highfidelity/hifi
# Prevent dragging and dropping files onto Create windows

Addresses: https://highfidelity.manuscript.com/f/cases/19084

Pre-Merge Test Plan:
1. Enter Interface in desktop mode into any domain
2. Open Create and verify that any file from file explorer being dragged onto either the Entity List or Entity Properties window shows a forbidden slash icon like below before releasing and does nothing once released
![hashicon](https://user-images.githubusercontent.com/31390055/49618113-222da100-f96c-11e8-8eb7-3454b639297d.png)
3. Verify step 2 on all tabs of the Entity Properties window
4. Verify step 2 on both PC and Mac (on Mac it is expected that there is no icon indicating you can't do that before releasing it)
